pages/resources/message.mdx

```diff
@@ -866,6 +868,7 @@ Note that when sending a message, you must provide a value for **at least one of
 | files[n]? ^1^               | file contents                                                                        | Contents of the file being sent (max 10)                                                                                                                                    |
 | payload_json? ^1^           | string                                                                               | JSON-encoded body of non-file params                                                                                                                                        |
 | attachments? ^1^            | array[partial [attachment](#attachment-object) object]                               | Partial attachment objects with `filename` and `description` (max 10)                                                                                                       |
+| poll?                       | [poll create request](/resources/poll#poll-create-request-object) object             | A poll!                                                                                                                                                                     |
```

Comment:
The above alert that mentions which fields are required should also be updated to mention `poll`. This alert is present in a couple other places in the docs as well.